### ISSUE TITLE
HOT Global Validators Team added by default to new projects when validation restricted to teams

### ISSUE THREAD

**Issue by SColchester** [hotosm/tasking-manager#6628]
**Problem**
There are lots of projects that make very slow progress / get stuck with validation because the project creator restricts validation to teams and then assigns only a small and inactive team of validators to validate the project. Project creators do not necessarily know that they can add any team to validate a project and they may also not be aware that the 'HOT Global Validators' team is an excellent group of very active validators. 

**Solution**
To ensure that more projects are available to validate by the HOT Global Validators team --> For any project in Tasking Manager, under the 'Permissions' section of the edit project page, if the project editor switches 'Validation' permissions to 'Only team members' (from the default of 'Only users with intermediate or advanced level') automatically populate 'HOT Global Validators' as a validation team. 

Only do this once, the first time that the project editor changes validation permissions to 'Only team members'. This way the project creator can add additional teams, or if they like, opt-out by manually removing the HOT Global Validator team. 

![2024-11-12 11_07_19-Edit project #17928 - HOT Tasking Manager — Mozilla Firefox](https://github.com/user-attachments/assets/7bcf62a3-c186-40bb-8d59-7bc00e5ebfac)

**Alternatives**
An alternative could be to add a fifth radio button under the 'Validation permissions' section which specifically calls out the HOT Global Validator Team, so could be an option for example stating: 'Only HOT validators'.

**Comment by ramyaragupathy** [comment]
@SColchester - thanks for bringing this up. @manjitapandey and I will verify the existing behaviour before we lock in a workflow.

cc @royallsilwallz

**Comment by martien-176** [comment]
I support the idea. 

I think it would be helpful if there is some additional description of what the Global Validator team is about. Project creators could not be aware of the value of the Globals. And "unkown, unloved" could cause them to skip the Globals if they can.

**Comment by SColchester** [comment]
Agree, not adding a radio button will be better, and yes - well spotted, for completeness it should automatically populate the HOT Global Validators team if either `only team members` or `only intermediate and advanced team members` is selected. 

I like the pop up on hover option for more info on the team, and would be ideal if this could be pulled directly from the team's description field. This is what that field looks like on the team 'manage' page. 

![Image](https://github.com/user-attachments/assets/c28ef371-4394-4988-af8c-27ce6abdb329)

**Comment by manjitapandey** [comment]
Shall we add the HOT global validator team if the project is already existing with permissions provided to only team member at the time of release? or will it be only for new projects or when the project manager changes the permission after the change released in production. 

cc: @SColchester , @ramyaragupathy , @martien-176

**Comment by SColchester** [comment]
@manjitapandey good question, don't apply this to existing projects. Should only be applied going forward to new projects that are created after the release. Project creators need to see that the team has been added by default and this gives them the best chance of removing it if they think that is best.
